libs/wingc/src/type_check.rs

```diff
@@ -2129,38 +2185,50 @@ impl<'a> TypeChecker<'a> {
 							}
 						}
 					}
-					_ => {}
+					_ => {
+						panic!("Unexpected symbol kind: {:?} in class env", symbol)
+					}
 				}
 			}
 		}
 
 		return new_type;
 	}
 
-	fn expr_maybe_type(&mut self, expr: &Expr, env: &SymbolEnv, statement_idx: usize) -> Option<TypeRef> {
+	fn expr_maybe_type(
```

Comment:
Docstring (purpose of the method isn't clear from its name IMO)

Reply:
done.